UKHO.PeriodicOutputService/UKHO.BESS.API.FunctionalTests/FunctionalTests/ScsIntegration/ScsIntegrationTests.cs

```diff
@@ -20,23 +20,15 @@ public async Task OneTimeSetupAsync()
         [Test]
         public async Task WhenICallScsEssDataEndpointWithValidToken_ThenSuccessStatusCode200IsReturned()
```

Comment:
Change the test name to WhenICallEssDataEndpointWithValidToken_ThenSuccessStatusCode200IsReturned()

Reply:
Resolved